src/datatypes/src/schema/column_schema.rs

```diff
@@ -34,6 +35,8 @@ pub const COMMENT_KEY: &str = "greptime:storage:comment";
 const DEFAULT_CONSTRAINT_KEY: &str = "greptime:default_constraint";
 /// Key used to store fulltext options in arrow field's metadata.
 pub const FULLTEXT_KEY: &str = "greptime:fulltext";
+/// Value used to identify column data type from arrow field's metadata. It should be the same as `ConcreteDataType::json_datatype().name()`.
+pub const JSON_TYPE_VALUE: &str = "Json";
```

Comment:
nitpick: use the same const in both here and json type def(since in same crate anyway)?

Reply:
Reasonable to me :)